modules/core/src/main/java/org/locationtech/jts/geom/GeometryFactory.java

```diff
@@ -100,7 +101,8 @@ public GeometryFactory() {
 
   private static CoordinateSequenceFactory getDefaultCoordinateSequenceFactory()
   {
-    return CoordinateArraySequenceFactory.instance();
```

Comment:
This is a MAJOR change.  It is not desirable IMO.  

Instead, the `PackedCoordinateSequence `implementation should be fixed, but not used as the default.  It can be tested directly.

Reply:
You are quick assigning WON'T FIX.
The fact, that the PackedCoordinateSequenceFactory uses toCoordinateArray() to create a CoordinateSequence off another is a major bug.

WIthout fixing that shifting WKT/WKBReader -Writer is next to useless.

Reply:
Yes, agreed, this is a major bug and needs to be fixed.  The WON'T FIX was just to indicate current status - was not meant to be a permanent thumbs-down!

But it's awkward to avoid merging the GeometryFactory change (and if I hadn't noticed that it would have been a problem!).  If you revert that we can discuss how to test your fix without such a major change.